CHANGELOG.md

```diff
@@ -70,6 +70,9 @@
   When exported constructors return `Self`.
   [#3562](https://github.com/rustwasm/wasm-bindgen/pull/3562)
 
+* Made `wasm-bindgen` forwards-compatible with the standard C ABI.
+  [#3595](https://github.com/rustwasm/wasm-bindgen/pull/3595)
```

Comment:
I actually meant mentioning that we broke `WasmAbi` & co.
Anyway, we should mention both.